runners/katacoda/templates/createProject.md

```diff
@@ -0,0 +1,16 @@
+<%= text; %>
+
+## Setting up your java project
+
+<%= cdCommand; %>
+
+Navigate to the 'workspaces/main/' folder in your devonfw installation directory.
+`cd workspaces/main/`{{execute}}
+
+Now you can use devon to setup a <%= language; %>  project for you by executing the following command.
+`devon <%= language; %> create com.example.application.<%= name; %>`{{execute}}
+
+Switch into the newly create project directory.
+`cd <%= name; %>`{{execute}}
```

Comment:
not sure, whether this is a good idea for maintainability.
We also have a cd step as I have seen. So possibly, we should not walk to different folders when creating a new project